app/src/main/java/com/quran/labs/androidquran/service/AudioService.java

```diff
@@ -357,6 +357,12 @@ public int onStartCommand(Intent intent, int flags, int startId) {
     }
 
     final String action = intent.getAction();
+    if (ACTION_PLAYBACK.equals(action)) {
```

Comment:
just moving it up sooner - and start foreground even if we're already foreground (multiple calls are fine, but missing a call is asking for a crash).